manager/manager.go

```diff
@@ -75,11 +90,24 @@ func NewManager(opts Options) (*Manager, error) {
 	if opts.WriteKeypair == nil {
 		return nil, errors.New("WriteKeypair must be set")
 	}
+	if opts.MaxRequestsPerVolume == 0 {
+		opts.MaxRequestsPerVolume = 1
```

Comment:
After some discussion, defaulting to 1 makes sense here. Anything above 1 seems to me like a special case, or temporarily set for debugging purposes. I would expect this value to be typically bubbled up to an end user anyway, where an author can enforce their own defaulting using a CLI flag. An author not exposing this option to the user will likely  be opinionated about the value anyway and set it themselves.

We can revisit this later to decide if and how we can represent a "clean up after use", for the cases where the driver should delete the CR immediately after it has been signed and consumed. There is also the option to _never_ cleanup CRs. In these cases we can either decide to utilise special negative numbers like below, and/or make a breaking API change to change this field type to be a pointer.

```go
const MaxRequestPerVolumeCleanupAfterUse = -1
```